extensions/sql-migration/src/wizard/skuRecommendationPage.ts

```diff
@@ -317,6 +339,8 @@ export class SKURecommendationPage extends MigrationWizardPage {
 		};
 		const textValue: string = `${count} databases will be migrated`;
 		this._rbg.cards[0].descriptions[1].textValue = textValue;
+		this._rbg.cards[1].descriptions[1].textValue = textValue;
```

Comment:
What are these two cards? It would be useful to add a comment - or find a better way to identify what the cards are and why you're setting the text values here

Reply:
@cssuh , I am assuming this change will be overwritten by your fixes. I will wait for you to remove this code.